kolibri/auth/test/test_permissions.py

```diff
@@ -0,0 +1,454 @@
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
I really like that the permissions test is thoroughly written with lots of cases, yet very simple to read and with no complicated call structure. Great work!